security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,37 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/co-management/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-4-0"]
+== OpenShift Compliance Operator 1.4.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.4.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:7658[RHBA-2023:7658 - OpenShift Compliance Operator bug fix and enhancement update]
+
+[id="compliance-operator-1-4-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* Users can now pause scan schedules by setting the `ScanSetting.suspend` attribute to `True`. This allows users to suspend a scan schedule and reactivate it without the need to delete and re-create the `ScanSettingBinding`. This simplifies pausing scan schedules during maintenance periods. link:https://issues.redhat.com/browse/CMP-2123[CMP-2123]
+
+* Compliance Operator now supports an optional `version` attribute on `Profile` custom resources. link:https://issues.redhat.com/browse/CMP-2125[CMP-2125]
+
+* Compliance Operator now supports profile names in `ComplianceRules`. link:https://issues.redhat.com/browse/CMP-2126[CMP-2126]
+
+* Compliance Operator compatibility with improved cronjob API improvements is available in this release. link:https://issues.redhat.com/browse/CMP-2310[CMP-2310]
+
+[id="compliance-operator-1-4-0-bug-fixes"]
+=== Bug fixes
+
+* With this update, rprivate default mount propagation is now handled correctly for root volume mounts of pods that rely on multipathing. link:https://issues.redhat.com/browse/OCPBUGS-17494[OCPBUGS-17494]
+
+* RHCOS audit rule failure locks are now applied correctly after auto-remediation. link:https://issues.redhat.com/browse/OCPBUGS-24594[OCPBUGS-24594]
```

Comment:
`RHCOS` can be an attribute:
{op-system-first} Red Hat Enterprise Linux CoreOS (RHCOS)
{op-system} RHCOS

Reply:
Done, thank you!